### ISSUE TITLE
Embers recipes not removed from JEI

### ISSUE THREAD

**Issue by bookerthegeek** [jaredlll08/ModTweaker#477]
#### Issue Description:
Recipes for Embers Melter not being removed from JEI.

#### What happens:
Removed Recipes no longer work, they are just not removed resulting in both new and old recipes to show up.

#### What you expected to happen:
For the removed recipe not to show up in JEI

#### Script used (Please Pastebin or gist your script, posting an unpasted or ungist'd script will automatically close this issue):
```
import mods.embers.Melter;

mods.embers.Melter.remove(<liquid:gold>);
mods.embers.Melter.addOreRecipe(<ore:oreGold>, <liquid:gold> * 48, false, false);
mods.embers.Melter.addOreRecipe(<ore:dustGold>, <liquid:gold> * 48, false, false);
mods.embers.Melter.addOreRecipe(<ore:nuggetGold>, <liquid:gold> * 16, false, false);
mods.embers.Melter.addOreRecipe(<ore:ingotGold>, <liquid:gold> * 144, false, false);
mods.embers.Melter.addOreRecipe(<ore:blockGold>, <liquid:gold> * 1296, false, false);
```

#### Minetweaker.log file:
The full log is [Here](https://gist.github.com/bookerthegeek/a3f455015564058c1566641ea8b86a50), below is the portion about the issue in question
```
[SERVER_ABOUT_TO_START][SERVER][INFO] Removing 3 Embers Melter Recipe(s) for nuggetGold, ingotGold, oreGold
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding 1 Embers Melter Recipe(s) for oreGold
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding 1 Embers Melter Recipe(s) for dustGold
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding 1 Embers Melter Recipe(s) for nuggetGold
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding 1 Embers Melter Recipe(s) for ingotGold
[SERVER_ABOUT_TO_START][SERVER][INFO] Adding 1 Embers Melter Recipe(s) for blockGold
```

____
#### Affected Versions (Do *not* use "latest"):

- Minecraft: 1.10.2
- Forge: 12.18.3.2254
- Crafttweaker: 1.10.2-3.0.24
- ModTweaker: 2.0.11
- MTLib: 1.0.2
- Embers: 0.104

#### Your most recent log file where the issue was present: 

[Here you go good sir](https://gist.github.com/bookerthegeek/2bcc220dd794c3de27340f46b160571e)

**Comment by Compegen** [comment]
Have you tried restarting your client after making changes to see if it is resolved?

**Comment by jaredlll08** [comment]
1.10.2 isn't supported anymore / Issue has been fixed in 1.12
